### DISCUSSION TITLE
Extension keeps "loading"

### DISCUSSION THREAD

**Discussion by mrspeedster** [AprilSylph/XKit-Rewritten#1031] [Help Desk]
I have the extension on Brave and it doesn't seem to load. I've deleted it and reinstalled it and it didn't help.

**Comment by marcustyphoon** [comment]
Worked for me in a quick test. The way you described "loading" makes me think you're describing some kind of loading indicator, but that I can think of, we don't really have one; what are you seeing? What Brave version do you have and on what platform?

**Reply by marcustyphoon** [reply]
Further questions:

Does the extension show up in **brave://extensions/**? Does the purple XKit Rewritten icon appear to the right of your URL bar in the browser UI? Does clicking on it make a settings menu appear? If so, and if you toggle on a feature with an obvious visual effect like Panorama, does it affect the page?

(If you're expecting something to happen in the Tumblr page itself when you install, that's not going to happen; by default all modifications are turned off, and the extension is configured from the menu in the browser UI!)

**Reply by AprilSylph** [reply]
(We do have one loading indicator in the control panel itself, which only appears while the features are being fetched, but I don't know what could cause a failure there)
